# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/strace/lxquotaver.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./resource.c.orig	2005-12-12 13:33:58.000000000 +0100
+++ ./resource.c	2005-12-12 13:37:51.000000000 +0100
@@ -32,6 +32,9 @@
 
 #include "defs.h"
 
+// strace doesn't know about the 2nd generation linux quota api
+#define _LINUX_QUOTA_VERSION 1
+
 #include <sys/resource.h>
 #ifdef LINUX
 #include <sys/times.h>
